configs/llm-360-amber-s3.yaml

```diff
@@ -0,0 +1,508 @@
+run_name: llm-360-amber-data-hyper-init-repro-3
+seed: 6198
+dry_run: false
+
+wandb:
+  name: ${run_name}
+  project: olmo-medium
+  group: ${run_name}
+  tags:
+    - epwalsh-amberish-stability
+
+model:
+  d_model: 4096
+  n_heads: 32
+  n_layers: 32
+  # mlp_ratio: 6
+  mlp_hidden_size: 22016
+  weight_tying: false
+  alibi: false
+  rope: true
+  flash_attention: true
+  attention_dropout: 0.0
+  attention_layer_norm: false
+  include_bias: false
+  block_type: sequential
+  layer_norm_type: rms
+  layer_norm_with_affine: false
+  layer_norm_eps: 1e-6
+  bias_for_layer_norm: false
+  attention_layer_norm_with_affine: false
+  activation_type: swiglu
+  residual_dropout: 0.0
+  embedding_dropout: 0.0
+  max_sequence_length: 2048
+  vocab_size: 32000
+  embedding_size: 32000
+  eos_token_id: 2
+  pad_token_id: 2
+  init_device: meta
+  init_fn: normal
+  init_std: 0.02
+
+compile: null
+
+optimizer:
+  name: adamw
+  learning_rate: 3.0e-4
+  weight_decay: 0.1
+  eps: 1e-8
+  decay_norm_and_bias: true
+  decay_embeddings: false
+  betas:
+  - 0.9
+  - 0.95
+  metrics_log_interval: 1
+
+scheduler:
+  name: cosine_with_warmup
+  units: tokens
+  t_warmup: 9175040000
+  t_max: 1.25e12
+  alpha_f: 0.1
+  warmup_min_lr: 0.0
+
+tokenizer:
+  identifier: tokenizers/nousresearch-llama-2-7b-hf.json
```

Comment:
I've since changed this to match ambers: 

huggyllama/llama-7b

This was the tokenizer I used for my pile experiment. While it's close, it's not identical to ambers.

Reply:
done